src/api-request-handler.ts

```diff
@@ -65,10 +64,9 @@ export class ApiRequestHandler {
                 case 400:
                     throw new InvalidParameterError(error);
                 case 401:
+                case 403:
                 case 404:
                     throw new RevAiApiError(error);
-                case 403:
```

Comment:
I have a feeling that we're using this scenario for multiple reasons.
Check [Python SDK](https://github.com/revdotcom/revai-python-sdk/blob/447ed2a1aa73289df5c1a44fbfea09319cb0b5ab/tests/helpers/errors.py#L30)

@kbridbur do you have any ideas on this?

Reply:
Submitting a request as a user with no credits does not actually return a 403 so wrapping it in a specific credits error is never used; that's why I removed it here.

Reply:
I guess that python sdk error should be removed as well -- it's never used

Reply:
I don't see anything in the code that returns 403. or `out-of-credit`. I think that was a previous implementation.
Let's keep this as is. Please post Python SDK ticket to make it consistent with your update